### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by atn34**
Ok this should be deployable. I'll start looking at patching in the PR's once this is merged. Let's keep all the routes in routes.py for now, moving them to different files caused 404's

### CODE REVIEW

File: src/routes.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+@namespace.route('/verify_email/<int:user_id>/<string:secret>')
+class VerifyEmail(Resource):
+    """Verifies an email address."""
+    @namespace.response(200, 'Success')
+    @namespace.response(401, 'Secret unrecognized')
+    def post(self, user_id, secret):
+        user = User.query.filter_by(id=user_id).first()
+        if user is not None and auth_util.verify_secret(
+                secret, user.email_verification_secret):
+            user.email_verified = True
```
Comment (whittlbc) [inline src/routes.py:85]: This will fail if `email_verified` isn't included in the user `__repr__`.

File: src/models.py [hunk 2]
```diff
@@ -58,7 +56,7 @@ def __init__(self, name, domains):
     self.name = name
     self.slug = slugify(name, separator='-', to_lower=True)
     self.domains = domains
+    self.email_verification_secret = auth_util.fresh_secret()
```
Comment (whittlbc) [inline src/models.py]: Nope

**Comment by whittlbc** [comment]
Weird that it caused 404s. If too much investigation time, will move until after deadline, but we should at least move the School api handler into `routes.py` rather than just deleting it.

File: src/models.py (hunk 2 above)
Comment (atn34) [reply src/models.py]: My bad, did not cherry pick 6ebd43c

File: src/routes.py (hunk 1 above)
Comment (atn34) [reply src/routes.py:85]: I'm not sure I follow. I believe `test_create_user_flow` demonstrates this working correctly.

That said, I've now cherry picked f/schools_fetch, which I believe makes whatever change you're talking about here.

**Pull request merged**